### PR TITLE
feat: ENG-6931 export FeaturedProductsCarouselSkeleton

### PR DESCRIPTION
**Pull request by jorgemoya**
Exports a Skeleton.

Still missing empty state.

Screenshot 2024-11-22 at 12 42 33 PM

### CODE REVIEW

File: apps/web/vibes/soul/primitives/products-carousel/index.tsx [hunk 1]
```diff
@@ -27,13 +27,10 @@ export function ProductsCarousel({
   return (
     <Carousel className={className}>
       <CarouselContent className="mb-10">
-        <Stream
-          value={streamableProducts}
-          fallback={<ProductsCarouselSkeleton className={className} message={emptyStateMessage} />}
-        >
+        <Stream value={streamableProducts} fallback={<ProductsCarouselSkeleton />}>
```
Comment (migueloller) [inline apps/web/vibes/soul/primitives/products-carousel/index.tsx:30]: I think the reason we were passing the class was to control with in Makeswift. Might need to rethink later by wrapping in div and passing class there

File: apps/web/vibes/soul/sections/featured-products-carousel/index.tsx [hunk 2]
```diff
@@ -45,3 +45,31 @@ export function FeaturedProductsCarousel({ title, description, cta, products }:
     </section>
   )
 }
+
+export function FeaturedProductsCarouselSkeleton({
+  title,
+  description,
+  cta,
```
Comment (migueloller) [inline apps/web/vibes/soul/sections/featured-products-carousel/index.tsx:52]: Are these going to always be available? i.e., not streamable?

**Review by migueloller — APPROVED** [review]

File: apps/web/vibes/soul/sections/featured-products-carousel/index.tsx (hunk 2 above)
Comment (jorgemoya) [reply apps/web/vibes/soul/sections/featured-products-carousel/index.tsx:52]: In Catalyst, yes, in general, maybe not?

File: apps/web/vibes/soul/primitives/products-carousel/index.tsx (hunk 1 above)
Comment (jorgemoya) [reply apps/web/vibes/soul/primitives/products-carousel/index.tsx:30]: It is weird that the `className` is being passed to the `Carousel` component but that same to the Skeleton (which is nested within `CarouselContent`). Seems like an oversight to me?

Comment (migueloller) [reply apps/web/vibes/soul/primitives/products-carousel/index.tsx:30]: That could be a mistake

**Pull request merged**